testing/test_terminal.py

```diff
@@ -1587,6 +1587,21 @@ def test_func():
                 assert "--calling--" not in s
                 assert "IndexError" not in s
 
+    def test_tb_line_show_capture(self, pytester: Pytester, option) -> None:
+        output_to_capture = "help! let me out!"
+        pytester.makepyfile(
+            f"""
+            import pytest
+            def test_fail():
+                print('{output_to_capture}')
+                assert False
+            """
+        )
+        result = pytester.runpytest("--tb=line")
+        result.stdout.fnmatch_lines(["*- Captured stdout call -*"])
+        s = result.stdout.str()
+        assert output_to_capture in s
```

Comment:
Perhaps just use the same `fnmatch_lines` call?


```suggestion
        result.stdout.fnmatch_lines([
            "*- Captured stdout call -*",
            output_to_capture
        ])
```

If you were having problems with that, I suspect it is the `!` in the `output_to_capture` string. Given it is not important to the test, I suggest just changing the string to `HELPLETMEOUT` or something like this.